zpm.c

```diff
@@ -210,7 +210,13 @@ int zpm_configuration_exists() {
 
 char* generate_repository_url(char* plugin_name) {
     char* url = malloc(PATH_MAX);
-    strcpy(url, "https://github.com/");
+    char* tmp = strstr(plugin_name, "/");
+
+    tmp = strstr(tmp + 1, "/");
+    if(!tmp)
```

Comment:
I'm afraid I do not understand the second statement in the list. is it relevant to that piece of code?